src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -29309,30 +29325,275 @@ class C<T>
             comp.VerifyDiagnostics();
         }
 
+        [Theory]
+        [InlineData("T")]
+        [InlineData("TNotNull")]
+        [WorkItem(39922, "https://github.com/dotnet/roslyn/issues/39922")]
+        public void EnforcedInMethodBody_MaybeNullWhen(string type)
+        {
+            var source = @"
+#nullable enable
+using System.Diagnostics.CodeAnalysis;
+
+class C<T, TClass, TNotNull>
+    where TClass : class
```

Comment:
>TClass [](start = 10, length = 6)

Consider deleting the TClass from this test.

Reply:
➡️ deleted extraneous type parameter